tests/installation/logs_from_installation_system.pm

```diff
@@ -45,11 +45,14 @@ sub run {
     # we require ssh installation anyway
     if (get_var('BACKEND', '') =~ /ipmi|spvm/) {
         use_ssh_serial_console;
-        # set serial console for xen
-        set_serial_console_on_vh('/mnt', '', 'xen') if (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen'));
-        set_serial_console_on_vh('/mnt', '', 'kvm') if (check_var('HOST_HYPERVISOR', 'kvm') || check_var('SYSTEM_ROLE', 'kvm'));
-        adjust_for_ipmi_xen('/mnt')                 if (get_var('REGRESSION') && (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen')));
-        set_pxe_efiboot('/mnt')                     if check_var('ARCH', 'aarch64');
+        # set serial console for xen and kvm of SLE hosts
+        # for openSUSE TW, it is set in other place where after kvm/xen patterns are installed
+        if (is_sle) {
+            set_serial_console_on_vh('/mnt', '', 'xen') if (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen'));
+            set_serial_console_on_vh('/mnt', '', 'kvm') if (check_var('HOST_HYPERVISOR', 'kvm') || check_var('SYSTEM_ROLE', 'kvm'));
+            adjust_for_ipmi_xen('/mnt')                 if (get_var('REGRESSION') && (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen')));
```

Comment:
Why  get_var('REGRESSION') ?

Reply:
REGRESSION might come from qem tests? It has been there before I added 'if(is_sle)'.